### ISSUE TITLE
Define a policy to generate a `default` form

### ISSUE THREAD

**Issue by relu91** [w3c/wot-binding-templates#121]
When generating an Exposed Thing Description, agents might need policies about how to create a default form given an affordance. Currently, in the scripting API we are leaving this aspect to the implementation (see step 6 of the [Expand ExposeThingInit algorithm](https://w3c.github.io/wot-scripting-api/#expand-an-exposedthinginit)). This might lead to fragmentation. Therefore we need to evaluate the possibility to describe such process per Protocol Binding Template, so that implementers have a reference algorithm on how to generate a well-known `form` from the specific binding template.  

Default mapping could be a starting point, what I think is missing is how the default URL would look like and the default values for every vocabulary term.

**Comment by egekorkan** [comment]
I agree that this is beneficial. I am not sure if this should be an assertive statement, i.e. given same partial TD to different implementations, one should get the same TD? We can of course add rules in some cases such as MQTT where the base uri should be used and should indicate the broker. For HTTP, I don't see a huge benefit to say assertively that properties should be exposed under `properties/myProp` rather than `p/myProp`

**Comment by egekorkan** [comment]
Actually, I think I forgot about the other parts of a form other than `href`. I think we can be assertive regarding everything but the `href`.
